### DISCUSSION TITLE
doubt regarding installation of squadds , i am getting the following error in my anaconda prompt

### DISCUSSION THREAD

**Discussion by ABHIRAM-KUNIYIL** [LFL-Lab/SQuADDS#25] [qiskit-fall-fest-2024]
error: subprocess-exited-with-error
  
  Building wheel for pandas (pyproject.toml) did not run successfully.
  exit code: 1
  
  [2514 lines of output]
  <string>:19: DeprecationWarning: pkg_resources is deprecated as an API. See https://setuptools.pypa.io/en/latest/pkg_resources.html
  running bdist_wheel
  running build
  running build_py
  creating build\lib.win-amd64-cpython-312\pandas
  copying pandas\conftest.py -> build\lib.win-amd64-cpython-312\pandas
  copying pandas\testing.py -> build\lib.win-amd64-cpython-312\pandas
  copying pandas\_typing.py -> build\lib.win-amd64-cpython-312\pandas
  copying pandas\_version.py -> build\lib.win-amd64-cpython-312\pandas
  copying pandas\__init__.py -> build\lib.win-amd64-cpython-312\pandas
  creating build\lib.win-amd64-cpython-312\pandas\api
  copying pandas\api\__init__.py -> build\lib.win-amd64-cpython-312\pandas\api
  creating build\lib.win-amd64-cpython-312\pandas\arrays

**Comment by shanto268** [comment]
Hi @ABHIRAM-KUNIYIL ,

As mentioned on Discord - this may be due to outdated C++ build tools on your system. Please update that and see if that resolves it.

Best,
Shanto
